### ISSUE TITLE
WooCommerce 3.1 CSV Import/Export

### ISSUE THREAD

**Issue by Jon007** [hyyan/woo-poly-integration#208]
This plugin doesn't do anything to break the new WooCommerce 3.1 CSV Import/Export but also doesn't do anything to help either.

What could be the issues/requirements to support it?

WooCommerce 3.1 CSV Import/Export:
- has no concept of Polylang language taxonomy:  custom meta and Product Attributes are supported but not arbitrary additional taxonomies, so an additional hook would be needed for this.
- import would not update related attributes in translated versions of products:  ie if you used import to update price, attributes etc of existing products, these would not be synchronised to the translated products so the data would become inconsistent.

**Comment by frottoli** [comment]
How to import a product with the standard csv importer ?
https://polylang.pro/doc/product-csv-importer-and-exporter/

I was thinking the link above was working but is not because the "language" field not exists

**Comment by Jon007** [comment]
language is a taxonomy not a field, check if the woocommerce importer supports taxonomies, a year ago when this was raised it did not, and this part was not implemented.

possible workarounds:
import products in base language only - they will be imported with no language set but you can use the polylang tool to assign them all to base language
or, use the polylang plugin which you have just linked to (which is not this plugin) and use their support channels

**Comment by frottoli** [comment]
yes but this plugin is not for free :) Ok now I try with taxonomies or with the workaround and I give you a feedback

**Comment by Jon007** [comment]
Import isn't synchronising meta on variations at the moment

**Comment by guillermo259** [comment]
> This plugin doesn't do anything to break the new WooCommerce 3.1 CSV Import/Export but also doesn't do anything to help either.
> 
> What could be the issues/requirements to support it?
> 
> WooCommerce 3.1 CSV Import/Export:
> 
> * has no concept of Polylang language taxonomy:  custom meta and Product Attributes are supported but not arbitrary additional taxonomies, so an additional hook would be needed for this.
> * import would not update related attributes in translated versions of products:  ie if you used import to update price, attributes etc of existing products, these would not be synchronised to the translated products so the data would become inconsistent.





I added a custom taxonomy to my woocommercer exporter so when I have to load the products I have a column called language and it adds each product to each language

**Comment by Jon007** [comment]
@guillermo259 you are welcome to contribute

**Comment by guillermo259** [comment]
I use this example https://github.com/woocommerce/woocommerce/wiki/Product-CSV-Importer-&-Exporter#adding-custom-export-columns-developers with this I add a new column to my export file the taxonomy "language"  and that show me in the export file which language has each product, and now I'm adding the taxonomy "post_translations" that show me the relationship between my products in each language
